unRAIDv6/dynamix.file.integrity.plg

```diff
@@ -319,6 +322,8 @@ if [[ -z $method ]]; then
   hash=sha256
 elif [[ $method == -b2 ]]; then
   hash=blake2
+elif [[ $method == -b3 ]]; then
```

Comment:
I don't know where the $method is coming from, so I am not sure if I broke this piece of code.

Reply:
Found and test it.